src/gym_electric_motor/physical_systems/electric_motors/six_phase_PMSM.py

```diff
@@ -0,0 +1,262 @@
+import math
+
+import numpy as np
+
+from .six_phase_motor import SixPhaseMotor
+
+
+class SixPhasePMSM(SixPhaseMotor):
+    """
+    =====================  ==========  ============= ===========================================
+    Motor Parameter        Unit        Default Value Description
+    =====================  ==========  ============= ===========================================
+    r_s                    Ohm         64.3e-3       Stator resistance
+    l_d                    H           125e-6        Direct axis inductance
+    l_q                    H           126e-6        Quadrature axis inductance
+    l_x                    H           39e-6         x-axis inductance
+    l_y                    H           35e-6         y-axis inductance
+    p                      1            5            Pole pair number
+    psi_PM                 Vs          4.7e-3        flux linkage of the permanent magnets
+    =====================  ==========  ============= ===========================================
+
```

Comment:
Small comment:
We should add the information here, where we have these parameters from:

The default parameters originate from:
L. Broghammer et al., "Reinforcement Learning Control of Six-Phase Permanent Magnet Synchronous Machines," 2023 13th International Electric Drives Production Conference (EDPC), Regensburg, Germany, 2023, pp. 1-8, doi: 10.1109/EDPC60603.2023.10372153.